src/test/java/bio/terra/service/snapshot/SnapshotDaoTest.java

```diff
@@ -231,50 +238,46 @@ public void snapshotEnumerateTest() throws Exception {
                 // dataset name or created_date. add a suffix to filter on for the even snapshots
                 .description(UUID.randomUUID().toString() + ((i % 2 == 0) ? "==foo==" : ""));
             String flightId = "snapshotEnumerateTest_flightId";
-            snapshotId = UUID.randomUUID();
+            UUID snapId = UUID.randomUUID();
```

Comment:
```suggestion
            UUID tmpSnapshotId = UUID.randomUUID();
```

Nit: Maybe something like this?